SharpPcap/LibPcap/PcapDevice.cs

```diff
@@ -586,6 +586,51 @@ public virtual void SendPacket(byte[] p, int size)
         /// <param name="size">The number of bytes to send</param>
         public abstract void SendPacket(ReadOnlySpan<byte> p);
 
+        #region Timestamp
+        public virtual TimestampResolution TimestampPrecision
+        {
+            get => (TimestampResolution)LibPcapSafeNativeMethods.pcap_get_tstamp_precision(PcapHandle);
+            set => LibPcapSafeNativeMethods.pcap_set_tstamp_precision(PcapHandle, (int)value);
```

Comment:
@chmorgan Are you sure this setter actually works?
`pcap_set_tstamp_precision` can not be called on an activated pcap handle.

Reply:
yeah that's probably broken, let me remove and comment on usage